选修课程/计算机视觉/HITCV_EXP/exp1/exp1-hand.py

```
@@ -0,0 +1,591 @@
+    kernel = np.array([[1, 4, 6, 4, 1],
+                       [4, 16, 24, 16, 4],
+                       [6, 24, 36, 24, 6],
+                       [4, 16, 24, 16, 4],
+                       [1, 4, 6, 4, 1]], dtype=np.float32) / 256.0
+    blurred = my_filter2D(image.astype(np.float32), kernel, padding='same')
+    # 下采样：每隔一行一列取一个像素
+    down = blurred[::2, ::2]
+    return np.clip(down, 0, 255).astype(np.uint8)
+
+def my_pyrUp(image):
+    """上采样：插零 + 高斯模糊（使用相同高斯核）"""
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)